.circleci/run_tests.sh

```diff
@@ -41,7 +41,7 @@ if [[ $num_studies > 0 ]]; then
   echo $list_csv
 
   test_reports_location="$HOME/repo/test-reports"
-  validation_command="$HOME/repo/cbioportal/core/src/main/scripts/importer/./validateStudies.py -d $HOME/repo/ -l $list_csv -p $HOME/repo/.circleci/portalinfo -html $test_reports_location --strict_mutation_checks"
```

Comment:
@sandertan maybe `--strict_mutation_checks` can be ON when study id contains `tcga`?